core/controllers/reader.py

```diff
@@ -203,7 +292,7 @@ def get(self, exploration_id):
                 feconf.ACTIVITY_TYPE_EXPLORATION, exploration_id):
             raise self.PageNotFoundException
 
-        is_iframed = (self.request.get('iframed') == 'true')
+        is_iframed = False
```

Comment:
Don't think you need this here -- just write False directly for the 'iframed' property in self.values.update.